server/src/main/resources/META-INF/services/org.apache.iotdb.db.mqtt.PayloadFormatter

```diff
@@ -0,0 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+org.apache.iotdb.db.mqtt.JSONPayloadFormatter
```

Comment:
Hi jialin, your json format is incorrect so I can not follow what you are concerning ...

1. (minor) Json uses ":" rather than "=";
2. (major) Json does not support 
```
{
...
"value" : { 
  [1, 2], 
  [3, 4]
 }
}
```
As "[]" is an array and "{}" is an object, everything in "{}" should be key-value pair.
So, the correct json is:
```
{
	"device": "root.sg.d1",
	"timestamp": "150423423423",
	"measurements": ["s1", "s2"],
	"values": {
		"s1": [5.0, "4.0"],
		"s2": [6.0, "7.0"]
	}
}
```
In this case, what is your concern?

Reply:
Users can implement `PayloadFormatter` interface and change the config `mqtt_payload_formatter` if the default json formatter `JSONPayloadFormatter` don't match the need.

For the scenario you stated, user might implement a `BatchJSONPayloadFormatter` and named `batchjson`.

Reply:
I wonder could the server provide two format at the same time..

Reply:
I updated the code to make default `json` formatter support both two json format:
```json
 {
      "device":"root.sg.d1",
      "timestamp":1586076045524,
      "measurements":["s1","s2"],
      "values":[0.530635,0.530635]
 }
```
or
```json
{
      "device":"root.sg.d1",
      "timestamps":[1586076045524,1586076065526],
      "measurements":["s1","s2"],
      "values":[[0.530635,0.530635], [0.530655,0.530695]]
  }
```

Reply:
Also added unit tests and docs.